hibernate-core/src/test/java/org/hibernate/test/naturalid/immutable/IdentifierGeneratorWithNaturalIdCacheTest.java

```diff
@@ -30,6 +32,7 @@
 /**
  * @author Alex Burgel
  */
```

Comment:
The following would skip the test for Oracle12cDialect as well.

Instead I used:
`@RequiresDialectFeature( value = DialectChecks.SupportsIdentityColumns.class, jiraKey = "HHH-11330")`